OrbitLinuxTracing/UprobesFunctionCallManagerTest.cpp

```diff
@@ -13,8 +13,9 @@ TEST(UprobesFunctionCallManager, OneUprobe) {
   constexpr pid_t tid = 42;
   std::optional<FunctionCall> processed_function_call;
   UprobesFunctionCallManager function_call_manager;
+  perf_event_uprobe_regs registers;
 
-  function_call_manager.ProcessUprobes(tid, 100, 1);
+  function_call_manager.ProcessUprobes(tid, 100, 1, registers);
```

Comment:
Maybe add at least some assertions based on the new `registers` field?

Reply:
Done.